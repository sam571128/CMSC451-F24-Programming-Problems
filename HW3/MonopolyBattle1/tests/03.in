100 100 1000
88 31
21 1
59 39
60 59
70 69
66 64
89 25
34 32
59 17
13 97
37 85
74 34
3 7
2 64
82 35
56 20
77 72
49 78
60 26
89 57
83 45
49 65
92 37
95 20
17 48
73 26
79 73
94 77
18 65
19 9
66 97
39 84
92 33
84 31
87 8
52 28
35 82
89 94
21 20
69 77
93 40
59 43
31 42
76 23
95 88
78 78
61 7
89 97
29 39
81 80
84 99
57 51
89 27
95 39
21 96
50 26
3 24
43 96
81 19
32 98
6 39
16 79
37 45
57 82
44 13
56 10
31 98
77 70
93 80
84 81
55 8
77 11
88 80
81 42
85 32
59 45
75 30
48 41
12 95
55 35
14 40
18 28
84 34
35 45
75 54
31 15
7 29
66 96
43 77
90 84
100 6
76 41
66 6
10 87
98 14
75 73
58 34
82 86
36 5
3 37
