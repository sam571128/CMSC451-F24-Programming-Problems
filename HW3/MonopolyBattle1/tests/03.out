118 135 118 118 118 118 276 118 216 153 135 148 219 118 118 160 176 214 190 144 118 186 311 118 118 135 112 186 118 118 135 160 214 176 239 227 160 191 118 197 118 137 148 118 159 118 160 135 154 118 118 112 332 118 257 118 118 118 118 118 209 118 178 112 347 118 153 253 195 118 118 165 160 166 153 176 118 132 173 129 144 137 118 227 118 118 145 135 118 135 230 160 135 274 118 197 118 145 118 153 